upstream-tag-sync/src/index.ts

```diff
@@ -277,22 +288,26 @@ async function run(): Promise<void> {
 
 This PR was automatically created by the sync action.`;
 
-    const prNumber = await createPullRequest(
-        octokit,
-        targetOwner,
-        targetRepoName,
-        prTitle,
-        prBody,
-        branchName,
-        defaultBranch,
-        ['sync', syncLabel]
-    );
-
-    info(`Created PR #${prNumber} to sync with ${latestTag}`);
+  const prNumber = await createPullRequest(
+    octokit,
+    targetOwner,
+    targetRepoName,
+    prTitle,
+    prBody,
+    branchName,
+    defaultBranch,
+    ['sync', syncLabel]
+  );
+
+  info(`Created PR #` + prNumber.toString() + ` to sync with ${latestTag}`);
```

Comment:
ditto

Reply:
If you want we can mute tsc with:
```json
{
  "rules": {
    "@typescript-eslint/restrict-template-expressions": [
      "error",
      {
        "allowNumber": true
      }
    ]
  }
}
```
Wdyt?

Reply:
As @RudolfSchreier mentioned in the other thread I'm just removing the `  ...tseslint.configs.strictTypeChecked,` option in the eslint config.